task/net/src/server.rs

```diff
@@ -8,22 +8,37 @@ use crate::notifications;
 use crate::{idl, link_local_iface_addr, MacAddressBlock};
 
 #[cfg(feature = "vlan")]
-use crate::generated::VLAN_RANGE;
+use crate::generated::VLANS;
 
 use drv_stm32h7_eth as eth;
 use idol_runtime::{ClientError, RequestError};
+use ringbuf::{ringbuf, ringbuf_entry};
 use task_net_api::{
     KszError, KszMacTableEntry, LargePayloadBehavior, MacAddress,
     ManagementCounters, ManagementLinkStatus, MgmtError, PhyError, RecvError,
-    SendError, SocketName, UdpMetadata,
+    SendError, SocketName, TrustError, UdpMetadata,
 };
 
+#[allow(dead_code)]
+#[derive(Copy, Clone, Eq, PartialEq, counters::Count)]
+enum Trace {
+    #[count(skip)]
+    None,
+    SetTrust {
+        vid: u16,
+        t: VLanTrust,
+    },
+    TrustExpired(u16),
```

Comment:
As a person who mostly interprets other people's ringbufs, it would be great if all primitive fields like this had names (like it does for SetTrust above)

Reply:
Good idea, done in 6359a7b5